src/translations/v6/v6.ts

```diff
@@ -87,10 +87,12 @@ export const getV6Translations = (abstractSqlModel = v6AbstractSqlModel) => {
 			'depends on-application': ['Cast', ['Null'], 'Integer'],
 		},
 		'application-has-name': {
-			$toResource: `application-has-config var name$${toVersion}`,
+			// TODO: Figure out why toVersion doesn't work.
+			$toResource: `application-has-config var name$resin`,
 		},
 		'device-has-name': {
-			$toResource: `device-has-config var name$${toVersion}`,
+			// TODO: Figure out why toVersion doesn't work.
```

Comment:
It would be good to give a description of what about it doesn't work, I've seen these sorts of messages enough time in the past and been completely unable to figure out what doesn't work but also unsure if it's a edge case that I'm just failing to find or if it's actually fixed

Reply:
Opened a pinejs issue with all the details and referenced it in the comment.
See: https://github.com/balena-io/pinejs/issues/794